sycl/test/inline-asm/asm_if.cpp

```diff
@@ -0,0 +1,51 @@
+// UNSUPPORTED: cuda
+// REQUIRES: gpu,linux
+// RUN: %clangxx -fsycl %s -DINLINE_ASM -o %t.out
+// RUN: %t.out
```

Comment:
What happens if I run this on a machine without a GPU? Should this be like
```suggestion
// RUN: %GPU_RUN_PLACEHOLDER %t.out
```